### ISSUE TITLE
Confirmation Messages

### ISSUE THREAD

**Issue by gianlucafiore** [saltcorn/saltcorn#2089]
Improved confirmation messages, as they are currently simple, and with a single generic text in English that asks if, you are sure, to write verbatim what you want to report, and perhaps also with formulas and environment variables, before accepting the execution of an action. I also propose that it can be a slightly more aesthetic popup, and not with a simple browser alert.

**Comment by Ingenu1ty** [comment]
I agree, confirmation messages need more aesthetics and more UX features👍 

I think a good solution would be to simply give the ability to assign an arbitrary view as a "confirmation message". One might wonder: what about buttons? - I think we need to add a new built-in action type: “confirm ok”, “confirm cancel”, etc. - which will only work in confirmation messages. 

**OR:**
Just another viewType: “**confirmation message view**” - just like show/edit view, but with a limited set of buttons, etc. - I think this solution is even better than the first one...
